content/docs/accessibility.md

```diff
@@ -376,7 +376,7 @@ These are toolboxes filled with HTML attributes that are fully supported in JSX
 
 Each type of widget has a specific design pattern and is expected to function in a certain way by users and user agents alike:
 
-- [WAI-ARIA Authoring Practices - Design Patterns and Widgets](https://www.w3.org/TR/wai-aria-practices/#aria_ex)
+- [ARIA Authoring Practices Guide (APG) - Design Patterns and Examples](https://www.w3.org/WAI/ARIA/apg/patterns/)
```

Comment:
```suggestion
- [ARIA Authoring Practices - Design Patterns and Examples](https://www.w3.org/WAI/ARIA/apg/#aria_ex)
```

https://www.w3.org/WAI/ARIA/apg/#aria_ex

I think this would be the appropriate URL?

Reply:
That anchor does not seem to exist.

ARIA Authoring Practices has been reorganized and the some page content has been moved to another page.

The previous page can be found on InternetArchive. I thought the link I posted was appropriate as a destination for this content.
https://web.archive.org/web/20220516223228/https://www.w3.org/TR/wai-aria-practices/#aria_ex